packages/headless/src/api/knowledge/stream-answer-api.ts

```diff
@@ -101,25 +111,25 @@ export const updateCacheWithEvent = (
     : {};
 
   switch (message.payloadType) {
-    case 'genqa.headerMessageType':
+    case PAYLOAD_TYPES.HEADER:
```

Comment:
This is a personal opinion, but I don't see the benefit of the new `PAYLOAD_TYPES` constant: there already is a `PayloadType` type, which `message.payloadType` uses, so you already have intelisense, and there would be a TS error if you entered an invalid value in the switch-case.

I also don't think it makes the code more (or less) readable, so I'd tend to stick with what we had.